assets/stylesheet.css

```diff
@@ -54,6 +47,42 @@ section.module.content {
   width: 100%
 }
 
+.sponsor-area {
+  padding-top: 5%;
+  padding-bottom: 5%;
+}
+
+.sponsor-logo {
+  width: 16%;
+  float: left;
+  padding-right: 2%;
+}
+
+.host-area {
+  padding-bottom: 10%;
+}
+
+.bottom-space {
+  padding-bottom: 5%;
+}
+
+.quote {
+  color: white;
+  background-color: #70addb;
+  padding: 5px 5px 10px 10px;
+  border-radius: 10px;
+  margin-bottom: 0 !important;
```

Comment:
Yeah I just want this to get live. Because anything is better than our old website. :smile: 
I would like you people to confirm the content before, though.